vital/types/bounding_box.h

```diff
@@ -95,6 +95,32 @@ class bounding_box
     m_bbox =  Eigen::AlignedBox< T, 2 >( ul, lr );
   }
 
+  /**
+   * @brief Create default (invalid) box.
+   */
+  bounding_box()
+  {
+    reset();
```

Comment:
...pretty sure this is redundant?

Reply:
I am having the default ctor call the reset, so they are kept to be the same. 
In case anything new is added, folks should see to default variables in the reset, rather than the ctor (and maybe makine the ctor and reset method out of sync)

Reply:
Hmm, I suppose I can buy that... mostly it bugs me that `setEmpty` is getting called twice.

Reply:
Well, I could always just add a comment explaining this and to keep the reset in sync with the ctor

People read comments, right?